styles/dataDashboard.css

```diff
@@ -63,11 +63,11 @@ body {
 }
 
 /**.main-content**/
-
-.main-content {
-    background-color: lightslategray;
-    padding-left: 3rem;
-}
+  
+  .main-content {
+        background-color: lightslategray;
+        padding: 0 2rem;
+  }
```

Comment:
This is the small viewport size. I reduced the padding a tiny bit and applied it to both sides evenly.